==29738== Memcheck, a memory error detector
==29738== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==29738== Using Valgrind-3.13.0 and LibVEX; rerun with -h for copyright info
==29738== Command: /home/wasmer/repositories/wasmer/wasmer/target/debug/examples/compiler-cranelift
==29738== 
==29738== 
==29738== HEAP SUMMARY:
==29738==     in use at exit: 19,464 bytes in 78 blocks
==29738==   total heap usage: 529 allocs, 451 frees, 94,636 bytes allocated
==29738== 
==29738== 8 bytes in 1 blocks are still reachable in loss record 1 of 30
==29738==    at 0x4C31B0F: malloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==29738==    by 0xD5F7BB: alloc::alloc::alloc (alloc.rs:80)
==29738==    by 0xD605E3: <alloc::alloc::Global as core::alloc::AllocRef>::alloc (alloc.rs:174)
==29738==    by 0xD5F71C: alloc::alloc::exchange_malloc (alloc.rs:277)
==29738==    by 0xD30F82: new<alloc::sync::Arc<rayon_core::registry::Registry>> (boxed.rs:177)
==29738==    by 0xD30F82: rayon_core::util::leak (util.rs:5)
==29738==    by 0xD3627E: rayon_core::registry::set_global_registry::{{closure}}::{{closure}} (registry.rs:196)
==29738==    by 0xD6C2B8: core::result::Result<T,E>::map (result.rs:519)
==29738==    by 0xD36319: rayon_core::registry::set_global_registry::{{closure}} (registry.rs:195)
==29738==    by 0xD597D3: std::sync::once::Once::call_once::{{closure}} (once.rs:264)
==29738==    by 0x1C16D69: std::sync::once::Once::call_inner (once.rs:416)
==29738==    by 0xD59750: std::sync::once::Once::call_once (once.rs:264)
==29738==    by 0xD36226: rayon_core::registry::set_global_registry (registry.rs:194)
==29738== 
==29738== 32 bytes in 1 blocks are still reachable in loss record 2 of 30
==29738==    at 0x4C33B25: calloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==29738==    by 0x4E3F7F4: _dlerror_run (dlerror.c:140)
==29738==    by 0x4E3F165: dlsym (dlsym.c:70)
==29738==    by 0x1C2BC93: fetch (weak.rs:61)
==29738==    by 0x1C2BC93: get<unsafe extern "C" fn(*const libc::unix::linux_like::linux::gnu::b64::x86_64::pthread_attr_t) -> usize> (weak.rs:46)
==29738==    by 0x1C2BC93: min_stack_size (thread.rs:449)
==29738==    by 0x1C2BC93: std::sys::unix::thread::Thread::new (thread.rs:49)
==29738==    by 0xD65CBB: std::thread::Builder::spawn_unchecked (mod.rs:492)
==29738==    by 0xD66EEB: std::thread::Builder::spawn (mod.rs:386)
==29738==    by 0xD35D08: <rayon_core::registry::DefaultSpawn as rayon_core::registry::ThreadSpawn>::spawn (registry.rs:101)
==29738==    by 0xD36DEF: rayon_core::registry::Registry::new (registry.rs:260)
==29738==    by 0xD35F7D: rayon_core::registry::global_registry::{{closure}} (registry.rs:169)
==29738==    by 0xD36309: rayon_core::registry::set_global_registry::{{closure}} (registry.rs:195)
==29738==    by 0xD597D3: std::sync::once::Once::call_once::{{closure}} (once.rs:264)
==29738==    by 0x1C16D69: std::sync::once::Once::call_inner (once.rs:416)
==29738== 
==29738== 32 bytes in 1 blocks are still reachable in loss record 3 of 30
==29738==    at 0x4C33B25: calloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==29738==    by 0x56C48E1: __cxa_thread_atexit_impl (cxa_thread_atexit_impl.c:106)
==29738==    by 0x1BF821E: register_dtor (fast_thread_local.rs:33)
==29738==    by 0x1BF821E: try_register_dtor<core::cell::RefCell<core::option::Option<std::sys_common::thread_info::ThreadInfo>>> (local.rs:438)
==29738==    by 0x1BF821E: std::thread::local::fast::Key<T>::try_initialize (local.rs:424)
==29738==    by 0x1C1AA19: get<core::cell::RefCell<core::option::Option<std::sys_common::thread_info::ThreadInfo>>,fn() -> core::cell::RefCell<core::option::Option<std::sys_common::thread_info::ThreadInfo>>> (local.rs:410)
==29738==    by 0x1C1AA19: __getit (local.rs:175)
==29738==    by 0x1C1AA19: try_with<core::cell::RefCell<core::option::Option<std::sys_common::thread_info::ThreadInfo>>,closure-0,()> (local.rs:262)
==29738==    by 0x1C1AA19: with<core::cell::RefCell<core::option::Option<std::sys_common::thread_info::ThreadInfo>>,closure-0,()> (local.rs:239)
==29738==    by 0x1C1AA19: std::sys_common::thread_info::set (thread_info.rs:40)
==29738==    by 0xD66081: std::thread::Builder::spawn_unchecked::{{closure}} (mod.rs:473)
==29738==    by 0xD5A79E: core::ops::function::FnOnce::call_once{{vtable-shim}} (function.rs:233)
==29738==    by 0x1C2C009: call_once<(),FnOnce<()>> (boxed.rs:1081)
==29738==    by 0x1C2C009: call_once<(),alloc::boxed::Box<FnOnce<()>>> (boxed.rs:1081)
==29738==    by 0x1C2C009: std::sys::unix::thread::Thread::new::thread_start (thread.rs:87)
==29738==    by 0x52516DA: start_thread (pthread_create.c:463)
==29738==    by 0x57A271E: clone (clone.S:95)
==29738== 
==29738== 32 bytes in 1 blocks are still reachable in loss record 4 of 30
==29738==    at 0x4C33B25: calloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==29738==    by 0x56C48E1: __cxa_thread_atexit_impl (cxa_thread_atexit_impl.c:106)
==29738==    by 0xD9DFF1: std::thread::local::fast::Key<T>::try_register_dtor (local.rs:438)
==29738==    by 0xD9DEF2: std::thread::local::fast::Key<T>::try_initialize (local.rs:424)
==29738==    by 0xD9E096: std::thread::local::fast::Key<T>::get (local.rs:410)
==29738==    by 0xD9DDEC: crossbeam_epoch::default::HANDLE::__getit (local.rs:175)
==29738==    by 0xD47A94: std::thread::local::LocalKey<T>::try_with (local.rs:262)
==29738==    by 0xD4D422: crossbeam_epoch::default::with_handle (default.rs:43)
==29738==    by 0xD4D706: crossbeam_epoch::default::is_pinned (default.rs:30)
==29738==    by 0xD4352C: crossbeam_deque::deque::Stealer<T>::steal (deque.rs:619)
==29738==    by 0xD390D2: rayon_core::registry::WorkerThread::steal::{{closure}} (registry.rs:779)
==29738==    by 0xD344BC: core::iter::traits::iterator::Iterator::find_map::check::{{closure}} (iterator.rs:2258)
==29738== 
==29738== 64 bytes in 4 blocks are still reachable in loss record 5 of 30
==29738==    at 0x4C31B0F: malloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==29738==    by 0xD5F7BB: alloc::alloc::alloc (alloc.rs:80)
==29738==    by 0xD605E3: <alloc::alloc::Global as core::alloc::AllocRef>::alloc (alloc.rs:174)
==29738==    by 0xD5F71C: alloc::alloc::exchange_malloc (alloc.rs:277)
==29738==    by 0xD44C66: new<crossbeam_deque::deque::Buffer<rayon_core::job::JobRef>> (boxed.rs:177)
==29738==    by 0xD44C66: <T as crossbeam_epoch::atomic::Pointable>::init (atomic.rs:188)
==29738==    by 0xD61513: crossbeam_epoch::atomic::Owned<T>::init (atomic.rs:861)
==29738==    by 0xD61663: crossbeam_epoch::atomic::Atomic<T>::init (atomic.rs:309)
==29738==    by 0xD61633: crossbeam_epoch::atomic::Atomic<T>::new (atomic.rs:294)
==29738==    by 0xD43366: crossbeam_deque::deque::Worker<T>::new_lifo (deque.rs:253)
==29738==    by 0xD3726E: rayon_core::registry::Registry::new::{{closure}} (registry.rs:229)
==29738==    by 0xD6FF7C: core::iter::adapters::map_fold::{{closure}} (mod.rs:833)
==29738==    by 0xD40BCD: core::iter::traits::iterator::Iterator::fold (iterator.rs:2022)
==29738== 
==29738== 64 bytes in 4 blocks are still reachable in loss record 6 of 30
==29738==    at 0x4C31B0F: malloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==29738==    by 0x1C2BBEC: alloc (alloc.rs:80)
==29738==    by 0x1C2BBEC: alloc (alloc.rs:174)
==29738==    by 0x1C2BBEC: exchange_malloc (alloc.rs:277)
==29738==    by 0x1C2BBEC: std::sys::unix::thread::Thread::new (thread.rs:44)
==29738==    by 0xD65CBB: std::thread::Builder::spawn_unchecked (mod.rs:492)
==29738==    by 0xD66EEB: std::thread::Builder::spawn (mod.rs:386)
==29738==    by 0xD35D08: <rayon_core::registry::DefaultSpawn as rayon_core::registry::ThreadSpawn>::spawn (registry.rs:101)
==29738==    by 0xD36DEF: rayon_core::registry::Registry::new (registry.rs:260)
==29738==    by 0xD35F7D: rayon_core::registry::global_registry::{{closure}} (registry.rs:169)
==29738==    by 0xD36309: rayon_core::registry::set_global_registry::{{closure}} (registry.rs:195)
==29738==    by 0xD597D3: std::sync::once::Once::call_once::{{closure}} (once.rs:264)
==29738==    by 0x1C16D69: std::sync::once::Once::call_inner (once.rs:416)
==29738==    by 0xD59750: std::sync::once::Once::call_once (once.rs:264)
==29738==    by 0xD36226: rayon_core::registry::set_global_registry (registry.rs:194)
==29738== 
==29738== 72 bytes in 1 blocks are still reachable in loss record 7 of 30
==29738==    at 0x4C31B0F: malloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==29738==    by 0x181E37B: alloc::alloc::alloc (alloc.rs:80)
==29738==    by 0x181E4A3: <alloc::alloc::Global as core::alloc::AllocRef>::alloc (alloc.rs:174)
==29738==    by 0x181E2DC: alloc::alloc::exchange_malloc (alloc.rs:277)
==29738==    by 0x1821197: std::sync::rwlock::RwLock<T>::new (rwlock.rs:134)
==29738==    by 0x18214CC: <std::sync::rwlock::RwLock<T> as core::default::Default>::default (rwlock.rs:451)
==29738==    by 0x1812E40: __static_ref_initialize (frame_info.rs:28)
==29738==    by 0x1812E40: core::ops::function::FnOnce::call_once (function.rs:233)
==29738==    by 0x184AB5B: lazy_static::lazy::Lazy<T>::get::{{closure}} (inline_lazy.rs:31)
==29738==    by 0x18494D4: std::sync::once::Once::call_once::{{closure}} (once.rs:264)
==29738==    by 0x1C16D69: std::sync::once::Once::call_inner (once.rs:416)
==29738==    by 0x1849450: std::sync::once::Once::call_once (once.rs:264)
==29738==    by 0x180E268: get<std::sync::rwlock::RwLock<wasmer_engine::trap::frame_info::GlobalFrameInfo>,fn() -> std::sync::rwlock::RwLock<wasmer_engine::trap::frame_info::GlobalFrameInfo>> (inline_lazy.rs:30)
==29738==    by 0x180E268: __stability (lib.rs:142)
==29738==    by 0x180E268: <wasmer_engine::trap::frame_info::FRAME_INFO as core::ops::deref::Deref>::deref (lib.rs:144)
==29738== 
==29738== 160 bytes in 4 blocks are still reachable in loss record 8 of 30
==29738==    at 0x4C31B0F: malloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==29738==    by 0xD5F7BB: alloc::alloc::alloc (alloc.rs:80)
==29738==    by 0xD605E3: <alloc::alloc::Global as core::alloc::AllocRef>::alloc (alloc.rs:174)
==29738==    by 0xD5F71C: alloc::alloc::exchange_malloc (alloc.rs:277)
==29738==    by 0xD643CF: std::sync::mutex::Mutex<T>::new (mutex.rs:223)
==29738==    by 0xD77BE0: rayon_core::latch::LockLatch::new (latch.rs:230)
==29738==    by 0xD3801C: rayon_core::registry::ThreadInfo::new (registry.rs:583)
==29738==    by 0xD5A6B1: core::ops::function::FnMut::call_mut (function.rs:154)
==29738==    by 0xD6FC93: core::iter::adapters::map_fold::{{closure}} (mod.rs:833)
==29738==    by 0xD53633: core::iter::traits::iterator::Iterator::fold (iterator.rs:2022)
==29738==    by 0xD71548: <core::iter::adapters::Map<I,F> as core::iter::traits::iterator::Iterator>::fold (mod.rs:873)
==29738==    by 0xD6F449: core::iter::traits::iterator::Iterator::for_each (iterator.rs:658)
==29738== 
==29738== 160 bytes in 4 blocks are still reachable in loss record 9 of 30
==29738==    at 0x4C31B0F: malloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==29738==    by 0xD5F7BB: alloc::alloc::alloc (alloc.rs:80)
==29738==    by 0xD605E3: <alloc::alloc::Global as core::alloc::AllocRef>::alloc (alloc.rs:174)
==29738==    by 0xD5F71C: alloc::alloc::exchange_malloc (alloc.rs:277)
==29738==    by 0xD643CF: std::sync::mutex::Mutex<T>::new (mutex.rs:223)
==29738==    by 0xD77BE0: rayon_core::latch::LockLatch::new (latch.rs:230)
==29738==    by 0xD38034: rayon_core::registry::ThreadInfo::new (registry.rs:584)
==29738==    by 0xD5A6B1: core::ops::function::FnMut::call_mut (function.rs:154)
==29738==    by 0xD6FC93: core::iter::adapters::map_fold::{{closure}} (mod.rs:833)
==29738==    by 0xD53633: core::iter::traits::iterator::Iterator::fold (iterator.rs:2022)
==29738==    by 0xD71548: <core::iter::adapters::Map<I,F> as core::iter::traits::iterator::Iterator>::fold (mod.rs:873)
==29738==    by 0xD6F449: core::iter::traits::iterator::Iterator::for_each (iterator.rs:658)
==29738== 
==29738== 160 bytes in 4 blocks are still reachable in loss record 10 of 30
==29738==    at 0x4C31B0F: malloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==29738==    by 0xD5F7BB: alloc::alloc::alloc (alloc.rs:80)
==29738==    by 0xD605E3: <alloc::alloc::Global as core::alloc::AllocRef>::alloc (alloc.rs:174)
==29738==    by 0xD5F71C: alloc::alloc::exchange_malloc (alloc.rs:277)
==29738==    by 0xD643CF: std::sync::mutex::Mutex<T>::new (mutex.rs:223)
==29738==    by 0xD67D43: <std::sync::mutex::Mutex<T> as core::default::Default>::default (mutex.rs:446)
==29738==    by 0xD52A2E: <rayon_core::sleep::WorkerSleepState as core::default::Default>::default (mod.rs:55)
==29738==    by 0xD4FE5D: <crossbeam_utils::cache_padded::CachePadded<T> as core::default::Default>::default (cache_padded.rs:77)
==29738==    by 0xD512CA: rayon_core::sleep::Sleep::new::{{closure}} (mod.rs:67)
==29738==    by 0xD6FE40: core::iter::adapters::map_fold::{{closure}} (mod.rs:833)
==29738==    by 0xD409D8: core::iter::traits::iterator::Iterator::fold (iterator.rs:2022)
==29738==    by 0xD71619: <core::iter::adapters::Map<I,F> as core::iter::traits::iterator::Iterator>::fold (mod.rs:873)
==29738== 
==29738== 160 bytes in 4 blocks are still reachable in loss record 11 of 30
==29738==    at 0x4C31B0F: malloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==29738==    by 0x1BF95BE: alloc (alloc.rs:80)
==29738==    by 0x1BF95BE: alloc (alloc.rs:174)
==29738==    by 0x1BF95BE: exchange_malloc (alloc.rs:277)
==29738==    by 0x1BF95BE: new<()> (mutex.rs:223)
==29738==    by 0x1BF95BE: std::thread::Thread::new (mod.rs:1149)
==29738==    by 0xD65A5D: std::thread::Builder::spawn_unchecked (mod.rs:462)
==29738==    by 0xD66EEB: std::thread::Builder::spawn (mod.rs:386)
==29738==    by 0xD35D08: <rayon_core::registry::DefaultSpawn as rayon_core::registry::ThreadSpawn>::spawn (registry.rs:101)
==29738==    by 0xD36DEF: rayon_core::registry::Registry::new (registry.rs:260)
==29738==    by 0xD35F7D: rayon_core::registry::global_registry::{{closure}} (registry.rs:169)
==29738==    by 0xD36309: rayon_core::registry::set_global_registry::{{closure}} (registry.rs:195)
==29738==    by 0xD597D3: std::sync::once::Once::call_once::{{closure}} (once.rs:264)
==29738==    by 0x1C16D69: std::sync::once::Once::call_inner (once.rs:416)
==29738==    by 0xD59750: std::sync::once::Once::call_once (once.rs:264)
==29738==    by 0xD36226: rayon_core::registry::set_global_registry (registry.rs:194)
==29738== 
==29738== 160 bytes in 4 blocks are still reachable in loss record 12 of 30
==29738==    at 0x4C31B0F: malloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==29738==    by 0xD5F7BB: alloc::alloc::alloc (alloc.rs:80)
==29738==    by 0xD605E3: <alloc::alloc::Global as core::alloc::AllocRef>::alloc (alloc.rs:174)
==29738==    by 0xD5F71C: alloc::alloc::exchange_malloc (alloc.rs:277)
==29738==    by 0xD2F427: alloc::sync::Arc<T>::new (sync.rs:324)
==29738==    by 0xD65AFB: std::thread::Builder::spawn_unchecked (mod.rs:465)
==29738==    by 0xD66EEB: std::thread::Builder::spawn (mod.rs:386)
==29738==    by 0xD35D08: <rayon_core::registry::DefaultSpawn as rayon_core::registry::ThreadSpawn>::spawn (registry.rs:101)
==29738==    by 0xD36DEF: rayon_core::registry::Registry::new (registry.rs:260)
==29738==    by 0xD35F7D: rayon_core::registry::global_registry::{{closure}} (registry.rs:169)
==29738==    by 0xD36309: rayon_core::registry::set_global_registry::{{closure}} (registry.rs:195)
==29738==    by 0xD597D3: std::sync::once::Once::call_once::{{closure}} (once.rs:264)
==29738== 
==29738== 192 bytes in 4 blocks are still reachable in loss record 13 of 30
==29738==    at 0x4C31B0F: malloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==29738==    by 0x1C166B6: alloc (alloc.rs:80)
==29738==    by 0x1C166B6: alloc (alloc.rs:174)
==29738==    by 0x1C166B6: exchange_malloc (alloc.rs:277)
==29738==    by 0x1C166B6: std::sync::condvar::Condvar::new (condvar.rs:130)
==29738==    by 0xD77BFC: rayon_core::latch::LockLatch::new (latch.rs:231)
==29738==    by 0xD3801C: rayon_core::registry::ThreadInfo::new (registry.rs:583)
==29738==    by 0xD5A6B1: core::ops::function::FnMut::call_mut (function.rs:154)
==29738==    by 0xD6FC93: core::iter::adapters::map_fold::{{closure}} (mod.rs:833)
==29738==    by 0xD53633: core::iter::traits::iterator::Iterator::fold (iterator.rs:2022)
==29738==    by 0xD71548: <core::iter::adapters::Map<I,F> as core::iter::traits::iterator::Iterator>::fold (mod.rs:873)
==29738==    by 0xD6F449: core::iter::traits::iterator::Iterator::for_each (iterator.rs:658)
==29738==    by 0xD57021: <alloc::vec::Vec<T> as alloc::vec::SpecExtend<T,I>>::spec_extend (vec.rs:2140)
==29738==    by 0xD57CA4: <alloc::vec::Vec<T> as alloc::vec::SpecExtend<T,I>>::from_iter (vec.rs:2120)
==29738==    by 0xD58C46: <alloc::vec::Vec<T> as core::iter::traits::collect::FromIterator<T>>::from_iter (vec.rs:1995)
==29738== 
==29738== 192 bytes in 4 blocks are still reachable in loss record 14 of 30
==29738==    at 0x4C31B0F: malloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==29738==    by 0x1C166B6: alloc (alloc.rs:80)
==29738==    by 0x1C166B6: alloc (alloc.rs:174)
==29738==    by 0x1C166B6: exchange_malloc (alloc.rs:277)
==29738==    by 0x1C166B6: std::sync::condvar::Condvar::new (condvar.rs:130)
==29738==    by 0xD77BFC: rayon_core::latch::LockLatch::new (latch.rs:231)
==29738==    by 0xD38034: rayon_core::registry::ThreadInfo::new (registry.rs:584)
==29738==    by 0xD5A6B1: core::ops::function::FnMut::call_mut (function.rs:154)
==29738==    by 0xD6FC93: core::iter::adapters::map_fold::{{closure}} (mod.rs:833)
==29738==    by 0xD53633: core::iter::traits::iterator::Iterator::fold (iterator.rs:2022)
==29738==    by 0xD71548: <core::iter::adapters::Map<I,F> as core::iter::traits::iterator::Iterator>::fold (mod.rs:873)
==29738==    by 0xD6F449: core::iter::traits::iterator::Iterator::for_each (iterator.rs:658)
==29738==    by 0xD57021: <alloc::vec::Vec<T> as alloc::vec::SpecExtend<T,I>>::spec_extend (vec.rs:2140)
==29738==    by 0xD57CA4: <alloc::vec::Vec<T> as alloc::vec::SpecExtend<T,I>>::from_iter (vec.rs:2120)
==29738==    by 0xD58C46: <alloc::vec::Vec<T> as core::iter::traits::collect::FromIterator<T>>::from_iter (vec.rs:1995)
==29738== 
==29738== 192 bytes in 4 blocks are still reachable in loss record 15 of 30
==29738==    at 0x4C31B0F: malloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==29738==    by 0x1C166B6: alloc (alloc.rs:80)
==29738==    by 0x1C166B6: alloc (alloc.rs:174)
==29738==    by 0x1C166B6: exchange_malloc (alloc.rs:277)
==29738==    by 0x1C166B6: std::sync::condvar::Condvar::new (condvar.rs:130)
==29738==    by 0xD52A4A: <rayon_core::sleep::WorkerSleepState as core::default::Default>::default (mod.rs:57)
==29738==    by 0xD4FE5D: <crossbeam_utils::cache_padded::CachePadded<T> as core::default::Default>::default (cache_padded.rs:77)
==29738==    by 0xD512CA: rayon_core::sleep::Sleep::new::{{closure}} (mod.rs:67)
==29738==    by 0xD6FE40: core::iter::adapters::map_fold::{{closure}} (mod.rs:833)
==29738==    by 0xD409D8: core::iter::traits::iterator::Iterator::fold (iterator.rs:2022)
==29738==    by 0xD71619: <core::iter::adapters::Map<I,F> as core::iter::traits::iterator::Iterator>::fold (mod.rs:873)
==29738==    by 0xD6F27A: core::iter::traits::iterator::Iterator::for_each (iterator.rs:658)
==29738==    by 0xD573E7: <alloc::vec::Vec<T> as alloc::vec::SpecExtend<T,I>>::spec_extend (vec.rs:2140)
==29738==    by 0xD57C01: <alloc::vec::Vec<T> as alloc::vec::SpecExtend<T,I>>::from_iter (vec.rs:2120)
==29738==    by 0xD58BE5: <alloc::vec::Vec<T> as core::iter::traits::collect::FromIterator<T>>::from_iter (vec.rs:1995)
==29738== 
==29738== 192 bytes in 4 blocks are still reachable in loss record 16 of 30
==29738==    at 0x4C31B0F: malloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==29738==    by 0x1BF9622: alloc (alloc.rs:80)
==29738==    by 0x1BF9622: alloc (alloc.rs:174)
==29738==    by 0x1BF9622: exchange_malloc (alloc.rs:277)
==29738==    by 0x1BF9622: new (condvar.rs:130)
==29738==    by 0x1BF9622: std::thread::Thread::new (mod.rs:1150)
==29738==    by 0xD65A5D: std::thread::Builder::spawn_unchecked (mod.rs:462)
==29738==    by 0xD66EEB: std::thread::Builder::spawn (mod.rs:386)
==29738==    by 0xD35D08: <rayon_core::registry::DefaultSpawn as rayon_core::registry::ThreadSpawn>::spawn (registry.rs:101)
==29738==    by 0xD36DEF: rayon_core::registry::Registry::new (registry.rs:260)
==29738==    by 0xD35F7D: rayon_core::registry::global_registry::{{closure}} (registry.rs:169)
==29738==    by 0xD36309: rayon_core::registry::set_global_registry::{{closure}} (registry.rs:195)
==29738==    by 0xD597D3: std::sync::once::Once::call_once::{{closure}} (once.rs:264)
==29738==    by 0x1C16D69: std::sync::once::Once::call_inner (once.rs:416)
==29738==    by 0xD59750: std::sync::once::Once::call_once (once.rs:264)
==29738==    by 0xD36226: rayon_core::registry::set_global_registry (registry.rs:194)
==29738== 
==29738== 320 bytes in 4 blocks are still reachable in loss record 17 of 30
==29738==    at 0x4C31B0F: malloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==29738==    by 0x1BF9671: alloc (alloc.rs:80)
==29738==    by 0x1BF9671: alloc (alloc.rs:174)
==29738==    by 0x1BF9671: exchange_malloc (alloc.rs:277)
==29738==    by 0x1BF9671: new<std::thread::Inner> (sync.rs:324)
==29738==    by 0x1BF9671: std::thread::Thread::new (mod.rs:1145)
==29738==    by 0xD65A5D: std::thread::Builder::spawn_unchecked (mod.rs:462)
==29738==    by 0xD66EEB: std::thread::Builder::spawn (mod.rs:386)
==29738==    by 0xD35D08: <rayon_core::registry::DefaultSpawn as rayon_core::registry::ThreadSpawn>::spawn (registry.rs:101)
==29738==    by 0xD36DEF: rayon_core::registry::Registry::new (registry.rs:260)
==29738==    by 0xD35F7D: rayon_core::registry::global_registry::{{closure}} (registry.rs:169)
==29738==    by 0xD36309: rayon_core::registry::set_global_registry::{{closure}} (registry.rs:195)
==29738==    by 0xD597D3: std::sync::once::Once::call_once::{{closure}} (once.rs:264)
==29738==    by 0x1C16D69: std::sync::once::Once::call_inner (once.rs:416)
==29738==    by 0xD59750: std::sync::once::Once::call_once (once.rs:264)
==29738==    by 0xD36226: rayon_core::registry::set_global_registry (registry.rs:194)
==29738== 
==29738== 384 bytes in 1 blocks are still reachable in loss record 18 of 30
==29738==    at 0x4C31B0F: malloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==29738==    by 0xDC695B: alloc::alloc::alloc (alloc.rs:80)
==29738==    by 0xDC7203: <alloc::alloc::Global as core::alloc::AllocRef>::alloc (alloc.rs:174)
==29738==    by 0xDBDD1A: alloc::raw_vec::finish_grow (raw_vec.rs:497)
==29738==    by 0xD7A9A0: alloc::raw_vec::RawVec<T,A>::grow_amortized (raw_vec.rs:429)
==29738==    by 0xD792F3: alloc::raw_vec::RawVec<T,A>::try_reserve (raw_vec.rs:310)
==29738==    by 0xD7CEF2: alloc::raw_vec::RawVec<T,A>::reserve (raw_vec.rs:300)
==29738==    by 0xD55DD8: alloc::vec::Vec<T>::reserve (vec.rs:505)
==29738==    by 0xD56F37: <alloc::vec::Vec<T> as alloc::vec::SpecExtend<T,I>>::spec_extend (vec.rs:2136)
==29738==    by 0xD57CA4: <alloc::vec::Vec<T> as alloc::vec::SpecExtend<T,I>>::from_iter (vec.rs:2120)
==29738==    by 0xD58C46: <alloc::vec::Vec<T> as core::iter::traits::collect::FromIterator<T>>::from_iter (vec.rs:1995)
==29738==    by 0xD6F0A8: core::iter::traits::iterator::Iterator::collect (iterator.rs:1671)
==29738== 
==29738== 416 bytes in 4 blocks are still reachable in loss record 19 of 30
==29738==    at 0x4C31B0F: malloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==29738==    by 0xD5F7BB: alloc::alloc::alloc (alloc.rs:80)
==29738==    by 0xD605E3: <alloc::alloc::Global as core::alloc::AllocRef>::alloc (alloc.rs:174)
==29738==    by 0xD5F71C: alloc::alloc::exchange_malloc (alloc.rs:277)
==29738==    by 0xD65C27: new<closure-0> (boxed.rs:177)
==29738==    by 0xD65C27: std::thread::Builder::spawn_unchecked (mod.rs:494)
==29738==    by 0xD66EEB: std::thread::Builder::spawn (mod.rs:386)
==29738==    by 0xD35D08: <rayon_core::registry::DefaultSpawn as rayon_core::registry::ThreadSpawn>::spawn (registry.rs:101)
==29738==    by 0xD36DEF: rayon_core::registry::Registry::new (registry.rs:260)
==29738==    by 0xD35F7D: rayon_core::registry::global_registry::{{closure}} (registry.rs:169)
==29738==    by 0xD36309: rayon_core::registry::set_global_registry::{{closure}} (registry.rs:195)
==29738==    by 0xD597D3: std::sync::once::Once::call_once::{{closure}} (once.rs:264)
==29738==    by 0x1C16D69: std::sync::once::Once::call_inner (once.rs:416)
==29738== 
==29738== 512 bytes in 1 blocks are still reachable in loss record 20 of 30
==29738==    at 0x4C33E76: memalign (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==29738==    by 0x4C33F91: posix_memalign (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==29738==    by 0x1C1C129: aligned_malloc (alloc.rs:95)
==29738==    by 0x1C1C129: alloc (alloc.rs:22)
==29738==    by 0x1C1C129: __rdl_alloc (alloc.rs:304)
==29738==    by 0xDC695B: alloc::alloc::alloc (alloc.rs:80)
==29738==    by 0xDC7203: <alloc::alloc::Global as core::alloc::AllocRef>::alloc (alloc.rs:174)
==29738==    by 0xDBDD1A: alloc::raw_vec::finish_grow (raw_vec.rs:497)
==29738==    by 0xD7AD00: alloc::raw_vec::RawVec<T,A>::grow_amortized (raw_vec.rs:429)
==29738==    by 0xD791F3: alloc::raw_vec::RawVec<T,A>::try_reserve (raw_vec.rs:310)
==29738==    by 0xD7CE52: alloc::raw_vec::RawVec<T,A>::reserve (raw_vec.rs:300)
==29738==    by 0xD55D78: alloc::vec::Vec<T>::reserve (vec.rs:505)
==29738==    by 0xD57327: <alloc::vec::Vec<T> as alloc::vec::SpecExtend<T,I>>::spec_extend (vec.rs:2136)
==29738==    by 0xD57C01: <alloc::vec::Vec<T> as alloc::vec::SpecExtend<T,I>>::from_iter (vec.rs:2120)
==29738== 
==29738== 640 bytes in 1 blocks are still reachable in loss record 21 of 30
==29738==    at 0x4C33E76: memalign (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==29738==    by 0x4C33F91: posix_memalign (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==29738==    by 0x1C1C129: aligned_malloc (alloc.rs:95)
==29738==    by 0x1C1C129: alloc (alloc.rs:22)
==29738==    by 0x1C1C129: __rdl_alloc (alloc.rs:304)
==29738==    by 0xD5F7BB: alloc::alloc::alloc (alloc.rs:80)
==29738==    by 0xD605E3: <alloc::alloc::Global as core::alloc::AllocRef>::alloc (alloc.rs:174)
==29738==    by 0xD5F71C: alloc::alloc::exchange_malloc (alloc.rs:277)
==29738==    by 0xD2F255: alloc::sync::Arc<T>::new (sync.rs:324)
==29738==    by 0xD369E7: rayon_core::registry::Registry::new (registry.rs:238)
==29738==    by 0xD35F7D: rayon_core::registry::global_registry::{{closure}} (registry.rs:169)
==29738==    by 0xD36309: rayon_core::registry::set_global_registry::{{closure}} (registry.rs:195)
==29738==    by 0xD597D3: std::sync::once::Once::call_once::{{closure}} (once.rs:264)
==29738==    by 0x1C16D69: std::sync::once::Once::call_inner (once.rs:416)
==29738== 
==29738== 640 bytes in 1 blocks are still reachable in loss record 22 of 30
==29738==    at 0x4C33E76: memalign (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==29738==    by 0x4C33F91: posix_memalign (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==29738==    by 0x1C1C129: aligned_malloc (alloc.rs:95)
==29738==    by 0x1C1C129: alloc (alloc.rs:22)
==29738==    by 0x1C1C129: __rdl_alloc (alloc.rs:304)
==29738==    by 0xDA285B: alloc::alloc::alloc (alloc.rs:80)
==29738==    by 0xDA2DA3: <alloc::alloc::Global as core::alloc::AllocRef>::alloc (alloc.rs:174)
==29738==    by 0xDA27BC: alloc::alloc::exchange_malloc (alloc.rs:277)
==29738==    by 0xD9EA35: alloc::sync::Arc<T>::new (sync.rs:324)
==29738==    by 0xD9EF79: <crossbeam_epoch::collector::Collector as core::default::Default>::default (collector.rs:32)
==29738==    by 0xD9EFA6: crossbeam_epoch::collector::Collector::new (collector.rs:40)
==29738==    by 0xD9D499: __static_ref_initialize (default.rs:13)
==29738==    by 0xD9D499: core::ops::function::FnOnce::call_once (function.rs:233)
==29738==    by 0xD9D305: lazy_static::lazy::Lazy<T>::get::{{closure}} (inline_lazy.rs:31)
==29738==    by 0xD9E5C4: std::sync::once::Once::call_once::{{closure}} (once.rs:264)
==29738== 
==29738== 808 bytes in 1 blocks are still reachable in loss record 23 of 30
==29738==    at 0x4C31B0F: malloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==29738==    by 0x181E37B: alloc::alloc::alloc (alloc.rs:80)
==29738==    by 0x181E4A3: <alloc::alloc::Global as core::alloc::AllocRef>::alloc (alloc.rs:174)
==29738==    by 0x181E2DC: alloc::alloc::exchange_malloc (alloc.rs:277)
==29738==    by 0x182C550: new<alloc::collections::btree::node::LeafNode<usize, wasmer_engine::trap::frame_info::ModuleInfoFrameInfo>> (boxed.rs:177)
==29738==    by 0x182C550: alloc::collections::btree::node::Root<K,V>::new_leaf (node.rs:157)
==29738==    by 0x1812E59: core::ops::function::FnOnce::call_once (function.rs:233)
==29738==    by 0x181F59C: core::option::Option<T>::get_or_insert_with (option.rs:853)
==29738==    by 0x1844E5E: alloc::collections::btree::map::BTreeMap<K,V>::ensure_root_is_owned (map.rs:2324)
==29738==    by 0x1845231: alloc::collections::btree::map::BTreeMap<K,V>::entry (map.rs:1061)
==29738==    by 0x18460A9: alloc::collections::btree::map::BTreeMap<K,V>::insert (map.rs:829)
==29738==    by 0x180DF7F: wasmer_engine::trap::frame_info::register (frame_info.rs:281)
==29738==    by 0xBD7354: <wasmer_engine_jit::artifact::JITArtifact as wasmer_engine::artifact::Artifact>::register_frame_info (artifact.rs:275)
==29738== 
==29738== 1,152 bytes in 4 blocks are possibly lost in loss record 24 of 30
==29738==    at 0x4C33B25: calloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==29738==    by 0x4013666: allocate_dtv (dl-tls.c:286)
==29738==    by 0x4013666: _dl_allocate_tls (dl-tls.c:530)
==29738==    by 0x5252227: allocate_stack (allocatestack.c:627)
==29738==    by 0x5252227: pthread_create@@GLIBC_2.2.5 (pthread_create.c:644)
==29738==    by 0x1C2BD3C: std::sys::unix::thread::Thread::new (thread.rs:66)
==29738==    by 0xD65CBB: std::thread::Builder::spawn_unchecked (mod.rs:492)
==29738==    by 0xD66EEB: std::thread::Builder::spawn (mod.rs:386)
==29738==    by 0xD35D08: <rayon_core::registry::DefaultSpawn as rayon_core::registry::ThreadSpawn>::spawn (registry.rs:101)
==29738==    by 0xD36DEF: rayon_core::registry::Registry::new (registry.rs:260)
==29738==    by 0xD35F7D: rayon_core::registry::global_registry::{{closure}} (registry.rs:169)
==29738==    by 0xD36309: rayon_core::registry::set_global_registry::{{closure}} (registry.rs:195)
==29738==    by 0xD597D3: std::sync::once::Once::call_once::{{closure}} (once.rs:264)
==29738==    by 0x1C16D69: std::sync::once::Once::call_inner (once.rs:416)
==29738== 
==29738== 1,520 bytes in 1 blocks are still reachable in loss record 25 of 30
==29738==    at 0x4C31B0F: malloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==29738==    by 0xD5F7BB: alloc::alloc::alloc (alloc.rs:80)
==29738==    by 0xD605E3: <alloc::alloc::Global as core::alloc::AllocRef>::alloc (alloc.rs:174)
==29738==    by 0xD5F71C: alloc::alloc::exchange_malloc (alloc.rs:277)
==29738==    by 0xD45041: new<crossbeam_deque::deque::Block<rayon_core::job::JobRef>> (boxed.rs:177)
==29738==    by 0xD45041: <crossbeam_deque::deque::Injector<T> as core::default::Default>::default (deque.rs:1187)
==29738==    by 0xD438CD: crossbeam_deque::deque::Injector<T>::new (deque.rs:1213)
==29738==    by 0xD367B3: rayon_core::registry::Registry::new (registry.rs:242)
==29738==    by 0xD35F7D: rayon_core::registry::global_registry::{{closure}} (registry.rs:169)
==29738==    by 0xD36309: rayon_core::registry::set_global_registry::{{closure}} (registry.rs:195)
==29738==    by 0xD597D3: std::sync::once::Once::call_once::{{closure}} (once.rs:264)
==29738==    by 0x1C16D69: std::sync::once::Once::call_inner (once.rs:416)
==29738==    by 0xD59750: std::sync::once::Once::call_once (once.rs:264)
==29738== 
==29738== 1,520 bytes in 1 blocks are still reachable in loss record 26 of 30
==29738==    at 0x4C31B0F: malloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==29738==    by 0xD5F7BB: alloc::alloc::alloc (alloc.rs:80)
==29738==    by 0xD605E3: <alloc::alloc::Global as core::alloc::AllocRef>::alloc (alloc.rs:174)
==29738==    by 0xD5F71C: alloc::alloc::exchange_malloc (alloc.rs:277)
==29738==    by 0xD45041: new<crossbeam_deque::deque::Block<rayon_core::job::JobRef>> (boxed.rs:177)
==29738==    by 0xD45041: <crossbeam_deque::deque::Injector<T> as core::default::Default>::default (deque.rs:1187)
==29738==    by 0xD438CD: crossbeam_deque::deque::Injector<T>::new (deque.rs:1213)
==29738==    by 0xD30E4C: rayon_core::job::JobFifo::new (job.rs:193)
==29738==    by 0xD3920A: rayon_core::registry::main_loop (registry.rs:806)
==29738==    by 0xD3583F: rayon_core::registry::ThreadBuilder::run (registry.rs:56)
==29738==    by 0xD35EF0: <rayon_core::registry::DefaultSpawn as rayon_core::registry::ThreadSpawn>::spawn::{{closure}} (registry.rs:101)
==29738==    by 0xD6E79F: std::sys_common::backtrace::__rust_begin_short_backtrace (backtrace.rs:130)
==29738==    by 0xD66D70: std::thread::Builder::spawn_unchecked::{{closure}}::{{closure}} (mod.rs:475)
==29738== 
==29738== 1,536 bytes in 4 blocks are still reachable in loss record 27 of 30
==29738==    at 0x4C33E76: memalign (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==29738==    by 0x4C33F91: posix_memalign (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==29738==    by 0x1C1C129: aligned_malloc (alloc.rs:95)
==29738==    by 0x1C1C129: alloc (alloc.rs:22)
==29738==    by 0x1C1C129: __rdl_alloc (alloc.rs:304)
==29738==    by 0xD5F7BB: alloc::alloc::alloc (alloc.rs:80)
==29738==    by 0xD605E3: <alloc::alloc::Global as core::alloc::AllocRef>::alloc (alloc.rs:174)
==29738==    by 0xD5F71C: alloc::alloc::exchange_malloc (alloc.rs:277)
==29738==    by 0xD2F075: alloc::sync::Arc<T>::new (sync.rs:324)
==29738==    by 0xD433FD: crossbeam_deque::deque::Worker<T>::new_lifo (deque.rs:250)
==29738==    by 0xD3726E: rayon_core::registry::Registry::new::{{closure}} (registry.rs:229)
==29738==    by 0xD6FF7C: core::iter::adapters::map_fold::{{closure}} (mod.rs:833)
==29738==    by 0xD40BCD: core::iter::traits::iterator::Iterator::fold (iterator.rs:2022)
==29738==    by 0xD717D3: <core::iter::adapters::Map<I,F> as core::iter::traits::iterator::Iterator>::fold (mod.rs:873)
==29738== 
==29738== 2,008 bytes in 1 blocks are still reachable in loss record 28 of 30
==29738==    at 0x4C31B0F: malloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==29738==    by 0xDA285B: alloc::alloc::alloc (alloc.rs:80)
==29738==    by 0xDA2DA3: <alloc::alloc::Global as core::alloc::AllocRef>::alloc (alloc.rs:174)
==29738==    by 0xDA27BC: alloc::alloc::exchange_malloc (alloc.rs:277)
==29738==    by 0xD9B179: new<crossbeam_epoch::sync::queue::Node<crossbeam_epoch::internal::SealedBag>> (boxed.rs:177)
==29738==    by 0xD9B179: <T as crossbeam_epoch::atomic::Pointable>::init (atomic.rs:188)
==29738==    by 0xD9AB25: crossbeam_epoch::atomic::Owned<T>::init (atomic.rs:861)
==29738==    by 0xD9AA75: crossbeam_epoch::atomic::Owned<T>::new (atomic.rs:846)
==29738==    by 0xD9B37B: crossbeam_epoch::sync::queue::Queue<T>::new (queue.rs:50)
==29738==    by 0xD96773: crossbeam_epoch::internal::Global::new (internal.rs:254)
==29738==    by 0xD9EF6B: <crossbeam_epoch::collector::Collector as core::default::Default>::default (collector.rs:32)
==29738==    by 0xD9EFA6: crossbeam_epoch::collector::Collector::new (collector.rs:40)
==29738==    by 0xD9D499: __static_ref_initialize (default.rs:13)
==29738==    by 0xD9D499: core::ops::function::FnOnce::call_once (function.rs:233)
==29738== 
==29738== 2,040 bytes in 1 blocks are still reachable in loss record 29 of 30
==29738==    at 0x4C31B0F: malloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==29738==    by 0xDA285B: alloc::alloc::alloc (alloc.rs:80)
==29738==    by 0xDA2DA3: <alloc::alloc::Global as core::alloc::AllocRef>::alloc (alloc.rs:174)
==29738==    by 0xDA27BC: alloc::alloc::exchange_malloc (alloc.rs:277)
==29738==    by 0xD961A9: new<crossbeam_epoch::internal::Local> (boxed.rs:177)
==29738==    by 0xD961A9: <T as crossbeam_epoch::atomic::Pointable>::init (atomic.rs:188)
==29738==    by 0xD9AAC5: crossbeam_epoch::atomic::Owned<T>::init (atomic.rs:861)
==29738==    by 0xD9AA25: crossbeam_epoch::atomic::Owned<T>::new (atomic.rs:846)
==29738==    by 0xD97012: crossbeam_epoch::internal::Local::register (internal.rs:392)
==29738==    by 0xD9EFCD: crossbeam_epoch::collector::Collector::register (collector.rs:45)
==29738==    by 0xD9DDB3: crossbeam_epoch::default::HANDLE::__init (default.rs:18)
==29738==    by 0xD9D478: core::ops::function::FnOnce::call_once (function.rs:233)
==29738==    by 0xDA2F4B: std::thread::local::lazy::LazyKeyInner<T>::initialize (local.rs:289)
==29738== 
==29738== 4,096 bytes in 4 blocks are still reachable in loss record 30 of 30
==29738==    at 0x4C31B0F: malloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==29738==    by 0xD5F7BB: alloc::alloc::alloc (alloc.rs:80)
==29738==    by 0xD605E3: <alloc::alloc::Global as core::alloc::AllocRef>::alloc (alloc.rs:174)
==29738==    by 0xD78FAD: alloc::raw_vec::RawVec<T,A>::allocate_in (raw_vec.rs:183)
==29738==    by 0xD7C220: alloc::raw_vec::RawVec<T,A>::with_capacity_in (raw_vec.rs:159)
==29738==    by 0xD7839E: alloc::raw_vec::RawVec<T>::with_capacity (raw_vec.rs:90)
==29738==    by 0xD53BAE: alloc::vec::Vec<T>::with_capacity (vec.rs:363)
==29738==    by 0xD41E10: crossbeam_deque::deque::Buffer<T>::alloc (deque.rs:45)
==29738==    by 0xD432E9: crossbeam_deque::deque::Worker<T>::new_lifo (deque.rs:248)
==29738==    by 0xD3726E: rayon_core::registry::Registry::new::{{closure}} (registry.rs:229)
==29738==    by 0xD6FF7C: core::iter::adapters::map_fold::{{closure}} (mod.rs:833)
==29738==    by 0xD40BCD: core::iter::traits::iterator::Iterator::fold (iterator.rs:2022)
==29738== 
==29738== LEAK SUMMARY:
==29738==    definitely lost: 0 bytes in 0 blocks
==29738==    indirectly lost: 0 bytes in 0 blocks
==29738==      possibly lost: 1,152 bytes in 4 blocks
==29738==    still reachable: 18,312 bytes in 74 blocks
==29738==         suppressed: 0 bytes in 0 blocks
==29738== 
==29738== For counts of detected and suppressed errors, rerun with: -v
==29738== ERROR SUMMARY: 1 errors from 1 contexts (suppressed: 0 from 0)
